Makefile

```diff
@@ -85,9 +85,24 @@ validate:
 	@pipenv run python -c 'import electionguard; print(electionguard.__package__ + " successfully imported")'
 
 # Test
+unit-tests:
+	@echo ✅ UNIT TESTS
+	pipenv run pytest tests/unit
+
+property-tests:
+	@echo ✅ PROPERTY TESTS
+	pipenv run pytest tests/test_decryption_mediator.py
```

Comment:
Is this one isolated due to the tests? Also linting is failing but it actually just looks like non used imports?

Reply:
If placed with the other property tests and run via "make test" everything passes. However, causes failures when "make coverage" runs. This issue does not happen when the test runs in isolation or ahead of other property tests. That is why I modified the property-test group this way.